arch/arch_test.go

```diff
@@ -40,6 +40,9 @@ func (s *archSuite) TestNormaliseArch(c *gc.C) {
 		{"ppc64le", "ppc64el"},
 		{"ppc64", "ppc64el"},
 		{"s390x", "s390x"},
+		{"risc", "riscv64"},
```

Comment:
per https://wiki.debian.org/Multiarch/Tuples#riscv64-linux-gnu, there is a ```riskv64``` we should ensure to capture as well

Reply:
Really? with a `k` ? I don't see it anywhere

Reply:
`riscv64` - autocorrect me

Reply:
the comment really applies to the MustCompile.